### DISCUSSION TITLE
i get a error when i install emqx-operator using helm,and i do not know why?

### DISCUSSION THREAD

**Discussion by Terry-Liu** [emqx/emqx-operator#1031] [Q&A] [answered]
the command like this:
helm upgrade --install emqx-operator emqx/emqx-operator --namespace emqx-operator-system   --version=2.0.2

![1712069057796](https://github.com/emqx/emqx-operator/assets/9688598/fb8dba0f-40e4-434e-b368-b9e261199f83)

**Comment by Rory-Z** [comment, marked as answer]
You need install cert-manager before install EMQX operator, check this document: https://docs.emqx.com/en/emqx-operator/latest/getting-started/getting-started.html#install-emqx-operator
